src/reducers/_zrxinstant.js

```diff
@@ -0,0 +1,46 @@
+const balanceManagerZrxInstantAddress =
+  process.env.REACT_APP_ZRX_INSTANT_ADDRESS;
+const balanceManagerZrxInstantRelayer =
+  process.env.REACT_APP_ZRX_INSTANT_RELAYER;
+
+// -- Constants ------------------------------------------------------------- //
+const ZRX_INSTANT_RENDER_MODAL_REQUEST =
+  'zrxinstant/ZRX_INSTANT_RENDER_MODAL_REQUEST';
+const ZRX_INSTANT_RENDER_MODAL_SUCCESS =
+  'zrxinstant/ZRX_INSTANT_RENDER_MODAL_SUCCESS';
+
+// -- Actions --------------------------------------------------------------- //
+export const zrxInstantInit = () => (dispatch, getState) => {
+  dispatch({ type: ZRX_INSTANT_RENDER_MODAL_REQUEST });
+  window.zeroExInstant.render({
+    orderSource: balanceManagerZrxInstantRelayer,
+    shouldDisablePushToHistory: true,
```

Comment:
As mentioned this is in review and should be pushed out before EOD. 

If you don't care about instant closing on "back" and re-rendering on "forward" though, you can just leave as is.